src/ucs/sys/topo.c

```diff
@@ -137,9 +137,7 @@ ucs_status_t ucs_topo_get_distance(ucs_sys_device_t device1,
                           path2, sizeof(path2));
 
     path_distance = ucs_path_calc_distance(path1, path2);
-    if (path_distance < 0) {
-        return (ucs_status_t)path_distance;
```

Comment:
this is needed to ignore devices that cannot be found in sysfs and return the default distance instead of failing; for example, during topo gtest we added a dummy device.

Reply:
maybe add this as a comment?